### ISSUE TITLE
Navigation Sidebar - consider refactor of UI

### ISSUE THREAD

**Issue by pacharanero** [rcpch/rcpch-audit-engine#806]
User of E12 will notice that we have a number of different navigation buttons in various different places throughout the E12 application.

I propose that a cleaner and possibly easier to maintain model would be to gather all page navigation, user options, settings, eytc together into a left sidebar, which would make navigation more consistent and possibly feel more modern for users.

Options available to different kinds of user would vary of course according to their user type/role, but bringing all this together into a single place would also mean that any HTMX or Django logic about which options are shown to which users would be also in just the one place.

### Items that would be candidates for moving to a left sidebar type list include:

**View Dashboard** (this option always present, returns to the main E12 top level page)

**Docs** (consider renaming to something that's not confusing - 'Docs' could refer to doctors, 'Documentation' is what nurses write before handover, 'Manual' might be a good fit, other options: 'Guidelines', 'Guidebook', 'Handbook', 'Flight Manual'...

**View all Epilepsy12 Staff** (consider renaming to 'Users' for internal consistency. Also remove 'Epilepsy12' because they aren't actually E12 Staff in the sense Amani/Niky are)
**Create new User**
**View all Children** (consider renaming to **_Cases_**, for internal consistency)
**Add new Child** (consider renaming to **_Case_**, for internal consistency)
**Jump to all KPI Metrics** (consider rewording - we don't need 'Jump')
**Publish** (publish what?)
**Django Admin** (Probably should say 'Django Admin' (or 'SuperUser Admin') just to highlight to people that it's not a place to mess about as you can break the site!)

**Accounts Settings**
**Logout**
**Reset Password** (currently on the top right menu)

### What **wouldn't** be needed in the left sidebar

**Home** (we are considering deprecating the Home Page entirely)

@AmaniKrayemRCPCH @nikyraja @sairapons @eatyourpeas @dc2007git any thoughts/views on this?

**Comment by eatyourpeas** [comment]
I agree very happy to revisit the user interface here. Perhaps something for the agenda on the 18/19th?

**Comment by nikyraja** [comment]
I like this idea! It'll feel similar to other webpages so the format would be easier to follow or users etc

These are my thoughts:
View Dashboard -> 'Organisation Dashboard'
Docs -> 'Guidance' would be good
View all Epilepsy12 Staff -> agree 'Users' will suffice 
Create new User --> I would argue that this would fit under the above and doesn't need to be separate
View all Children --> I prefer to this phrase. 'Cohort view' would work however may be confusing when there are multiple cohorts
Add new Child --> same as users, this would fit under the above
Jump to all KPI Metrics --> 'KPI Metrics'
Publish (publish what?) --> only for E12 staff, but again doesn't need a place on the side bar
Django Admin --> agree
Accounts Settings
Logout
Reset Password --> I think its fine to have this under account settings

The home page is useful when not logged in, however I agree that we don't need to direct users back to it when already logged in. 

Happy to discuss further on 18/19, or happy to see mock ups as you develop. Not a priority, but will improve user experience!

**Comment by mbarton** [comment]
[from sprint planning] consider collapsible nav to maximise space for the workflows (especially on bad screens!)

**Comment by mbarton** [comment]
Merging https://github.com/rcpch/rcpch-audit-engine/issues/1333 into this issue - it's not obvious on the dashboard what cohort you are currently looking at
